core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/BasicPathShadow2afp.java

```diff
@@ -479,4 +497,84 @@ private void crossSegmentShadowLine(
         }
     }
 
+    /** Passing params to methods via this class in order to reduce the line length.
+     * @author $Author: fevzi ozgul$
+     * @version $FullVersion$
+     * @mavengroupid $GroupId$
+     * @mavenartifactid $ArtifactId$
+     */
+    public  final  class CoordinatesParam {
+        private double x1;
+
+        private double y1;
+
+        private double x2;
+
+        private double y2;
+
+        private double curx;
+
+        private double cury;
+
+        /*Constructor.
+        *Constructor.
+        */
+        CoordinatesParam(double x1, double y1, double x2, double y2, double curx, double cury) {
+            this.x1 = x1;
+            this.y1 = y1;
+            this.x2 = x2;
+            this.y2 = y2;
+            this.curx = curx;
+            this.cury = cury;
+        }
+
+        public double getX1() {
+            return x1;
+        }
+
+        public void setX1(double x1) {
+            this.x1 = x1;
+        }
+
+        public double getY1() {
+            return y1;
```

Comment:
This access to the attribute is forbidden by the coding rules.

Compiler says:
```
     return x1;
	       ^^
Unqualified access to the field BasicPathShadow2afp.CoordinatesParam.x1 
```

You must add the `this.` prefix.